README.md

```diff
@@ -0,0 +1,57 @@
+# React News-explorer
+
+Welcome to the React Nwes explorer API frontend repository! This project is the frontend part of a full-stack web application built with React, Node.js, and MongoDB, which allows users to seacrh articles by keywords using "news-explorer api worldwide", save the articles and view images source of it with title and their descripoin and of course to surf to the article source page.
+
```

Comment:
I can sign in with invalid email. The button doesn't show my name because I didn't sign up first


![image](https://github.com/daniel-malka/news-explorer-frontend/assets/54933269/d448d81c-5d74-420f-bdce-559859b5a36c)